changelog.txt

```diff
@@ -1,3 +1,21 @@
+1.11.2 (2021-05-02)
+------------------------------------------------------------------------
+Change: [Win32] Limit hardware accelerated video driver to OpenGL 3.2 or higher (#9077)
+Change: More improvements to the GUI at different scales (#9075, #9102, #9107, #9133, #9174)
+Fix: [Network] Possible out-of-bounds memory access with certain password lengths (#9176)
+Fix: Crash when extra viewport with zero height has sign in view (#9175)
+Fix: Do not try to use regression AI as a random AI (#9164)
+Fix #6598: Crash when trying to connect to a server from a network game (#9163)
+Fix #9152: Screenshot success popup window was treated as an error (#9159)
+Fix: [Network,Win32] Network errors were handled badly (#9116)
+Fix: [Network] Savegame transfer could stall in certain circumstances (#9106)
+Fix #9097: [NewGRF] Cargo variable initial payment variable was being truncated (#9098)
+Fix: [NewGRF] Industry variable 66 and object variable 46 erroneously truncated the distance (#9088)
+Fix: [NewGRF] Industry variables 65 and 66 ignored the parameter, and always used the north tile (#9088)
+Fix: Do not include regression test AI in bundle (#9068)
+Fix #9062: Executable version was not updated (#9066, #9154)
+
```

Comment:
Have the following been left consciously out of the changelog?
* Fix: String validation could leave invalid Utf8 encoded strings (#9096)
* Fix #6598: [Network] Do not disconnect from network game before company number validation (#9163)
* Fix #6598: [Network] Fall back to main menu when joining a network game to prevent invalid game state (#9163)